docs/resources/app_v2.md

```diff
@@ -40,6 +57,7 @@ The following arguments are supported:
 * `wait` - (Optional) Wait until app is deployed. Default: `true` (bool)
 * `annotations` - (Optional/Computed) Annotations for the app v2 (map)
 * `labels` - (Optional/Computed) Labels for the app v2 (map)
+* `system_default_registry` - (Optional/Computed) Private registry for system images
```

Comment:
The description should match the schema and have the field type, so

`system_default_registry` - (Optional/Computed) System default registry providing images for app deployment (string)`

Reply:
Done